docs/docs/testing-your-assistant.mdx

```diff
@@ -231,11 +232,17 @@ as well as providing an overall average. You can save these reports as JSON file
 
 The confusion matrix shows which intents are mistaken for others.
 Any samples which have been incorrectly predicted are logged and saved to a file called `errors.json` for easier debugging.
+<div align="center">
+    <img alt="image" src={useBaseUrl("/img/intent_confusion_matrix_example.png")} width="70%" />
+</div>
 
 The histogram allows you to visualize the confidence for all predictions,
 with the correct and incorrect predictions being displayed by blue and red bars respectively.
 Improving the quality of your training data will move the blue histogram bars to the right and the
```

Comment:
This is a little bit confusing now that the blue and red bars don't really move the right or left. Maybe something like:

"Improving the quality of your training data will move the blue histogram bars up the plot and the red histogram bars down the plot."

Reply:
Done